podcasts/FeaturedSummaryViewController.swift

```diff
@@ -183,13 +181,26 @@ class FeaturedSummaryViewController: SimpleNotificationsViewController, GridLayo
         }
 
         dispatchGroup.notify(queue: DispatchQueue.main) { [weak self] in
+            guard let self else {
+                return
+            }
+
+            // Add featured podcasts
+            for (index, discoverPodcast) in podcastsToShow.enumerated() {
+                self.podcasts.append(discoverPodcast)
+
+                if index == (self.maxFeaturedItems - 1) { break }
+            }
```

Comment:
[Nit] I know this code isn't new, but this could be reduced down to a single line: 

```suggestion
            // Add featured podcasts
            self.podcasts = Array(podcastsToShow.prefix(self.maxFeaturedItems))
```